Day-10/todo-app/src/components/Todo.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import TodoForm from "./TodoForm";
+import TodoList from "./TodoList";
+
+const Todo = () => {
+  const [todos, setTodos] = useState([]);
+
+  const addTodo = (todo) => {
+    setTodos([todo, ...todos]);
```

Comment:
Here we are adding new element in the 1st position of array.
so you are doing that deliberately?

Reply:
Yes, when we add a new task, it should be placed first.